### DISCUSSION TITLE
Feature Request: add API like addAndGet()

### DISCUSSION THREAD

**Discussion by jianzhongli** [Tencent/MMKV#743] [Ideas]
Sometimes we would like to use MMKV as a multiprocess counter, it would be great if MMKV can add an API like [` addAndGet()`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/AtomicInteger.html#addAndGet-int-). 

Is this feature on the roadmap? Or will you accept a PR for this?

**Comment by lingol** [comment]
If we are about to do it, we have to do it right. That means a complete set of atomic operations such as [CAS](https://en.wikipedia.org/wiki/Compare-and-swap), not just `addAndGet()`.

**Comment by lingol** [comment]
And to your question, No, it's not on our roadmap. 
PR is welcome. But it has to be **feature complete** regarding atomic operations.
